types/configurable/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for configurable 0.0
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
Please fix the link to the project as well.

Reply:
my bad, did not notice it. Since this npm package does not have a link to its source, is npm link enough?